pulsar/schema.go

```diff
@@ -178,7 +180,7 @@ func (js *JSONSchema) Decode(data []byte, v interface{}) error {
 }
 
 func (js *JSONSchema) Validate(message []byte) error {
-	return js.Decode(message, nil)
+	return nil
```

Comment:
why do we need these changes?

Reply:
`Schema.Validate` method has never been called elsewhere before, but in actually, these methods will panic once they are called.
The second param cannot be nil.